src/cascadia/TerminalSettingsModel/ProfileCollectionEntry.h

```diff
@@ -0,0 +1,40 @@
+/*++
+Copyright (c) Microsoft Corporation
+Licensed under the MIT license.
+
+Module Name:
+- SeparatorEntry.h
+
+Abstract:
+- An entry in the "new tab" dropdown menu that represents some collection
+    of profiles. This is an abstract class that has concretisations like

```

Comment:
```suggestion
    of profiles. This is an abstract class that has subclasses like
```

or even just `concretizations`, just to make spellbot happy